core/src/css/ionic/utils.bundle.ionic.scss

```diff
@@ -1 +1,4 @@
 @forward "../../foundations/ionic.utility";
+
+// CSS Utils
+@import "../utils.bundle";
```

Comment:
I'm ok with this, as mostly this brings agnostic classes. But this also brings the padding utility-classes file, that bring padding values that might not match the ones from the new Design System and we are already creating those in the _/foundations/ionic.utility_

I would instead individually import the others, like:
@import "../float-elements";
@import "../text-alignment";
@import "../text-transformation";
@import "../flex-utils";
@import "../display";